docs/_threats/tr-7.md

```diff
@@ -1,14 +1,65 @@
 ---
-doc-status: Pre-Draft
+doc-status: Draft
 sequence: 7
 type:
   - Availability
 title: Availability of foundational model
 external-refs:
   - owasp-llm-top-10:
-      - "LLM04: Model Denial of Service"
+    - "LLM04: Model Denial of Service"
+  - FFIEC IT Handbook: Business Continuity Management 
+  - FFIEC Appendix J:
+    - "Strengthening the Resilience of Outsourced Technology Services"
 ---
 
-Hosted LLMs typical have usage limits, but often lack the sophisticated controls needed to manage this.
+RAG systems are proliferating due to the low barrier of entry compared to traditional chatbot technologies. RAG has applicability for internal users as well as supporting customer-related use cases.
 
-Given the commercial model of LLM services is typically on a per usage basis, there is the risk that significant resources can be expended via Chat App if uncontrolled access is allowed. e.g. DOS, injecting very large contexts, brute forcing to elicit very long responses, etc.
+Many LLMs require GPU compute infrastructure to provide an acceptable level of responsiveness. Furthermore, some of the best-performing LLMs are proprietary. Therefore, the path of least resistance is to call out to a Technology Service Provider (TSP).
+
+However, applications that index sensitive information may use on-prem GPUs because:
```

Comment:
at the moment, on-prem hosting of the foundational model is out of scope for the use case we are exploring - see: 

https://github.com/finos/ai-readiness/blob/main/docs/index.md?plain=1#L25